presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -217,6 +217,8 @@ void PrestoServer::run() {
 
   initializeCoordinatorDiscoverer();
   if (coordinatorDiscoverer_ != nullptr) {
+    const uint64_t frequencyMsMin = 100; // In ms
```

Comment:
Do we want to make this a pair of node level configs? Also name constexpr variable with prefix k -> s/frequencyMsMin/kFrequencyMsMin/